### ISSUE TITLE
Official Conda version of packages besides the pip ones

### ISSUE THREAD

**Issue by Rubiel1** [googleapis/google-cloud-python#10507]
Problem:
Jupyter notebooks are very convenient to use and are standard now a days to make test and demonstrations, and it is normal to install them with Anaconda. Installing gcp libraries with pip can lead to problems if the user is already using conda. So it would be nice to have listed here the official links to install those libraries using conda. If they don't exist, would it be worth creating them? I think so.

In the past I look for each library that I needed (sdk, big query, etc) and there are several links to install them using conda, but sometimes they are not compatible, like if I installed bigquery with one link, then I cannot install storage with ANY OTHER link available, so I need to remove my version of bigquery and find another link that is compatible with the storage link found online, etc. I wish I can say this is related to my version of python before installing, but those online links provide few information beforehand, since they are not official.  So I cannot know if they will be compatible or not until I install them.

If instead you provide the same support for conda as you provide for pip, we would benefit from installing with conda and avoiding losing time on installation, specially for new programmers.

**Comment by busunkim96** [comment]
CC @tswast

**Comment by tswast** [comment]
👍 I agree that it would help to be more deliberate about packaging all Google Cloud libraries for conda. Currently, I'm the primary maintainer of most of the `google-cloud-*` packages on conda-forge ([bigquery](https://github.com/conda-forge/google-cloud-bigquery-feedstock), [storage](https://github.com/conda-forge/google-cloud-storage-feedstock)

Currently, this requires some manual work to ensure the dependencies listed in `recipe/meta.yaml` stay in sync with `setup.py`. I would want more automation before signing up to maintain all that many more :-)

**Comment by tswast** [comment]
Aside: it appears this is the tracking bug for dependency synchronization https://github.com/regro/cf-scripts/issues/22

**Comment by medb** [comment]
It seems that Conda Forge channel has almost all google-cloud-* packages, but most of them are quite outdated. Are there plans to support then in Conda officially?

**Comment by parthea** [comment]
Closing as obsolete. There are currently no plans to offer official support for conda for all google cloud packages.
